README.md

```diff
@@ -1,2 +1,193 @@
-# hello_microverse
-Day 2 Microverse Program on github flow, use of linters and professional readme.md 
+<a name="readme-top"></a>
+<div align="center">
+  <img src="jojo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  # Hello Microverse  Project  Documentation
+</div>
+
+# 📗 Table of Contents
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
```

Comment:
Please all commented lines of codes should be removed from your 👍  project to ensure a legible and clean code

Reply:
Thanks, this has been addressed